crates/ark/src/lsp/completions/sources/composite/workspace.rs

```diff
@@ -87,15 +87,21 @@ fn completions_from_workspace(
                     return;
                 });
 
-                // add some metadata about where the completion was found
-                let mut path = path.to_str().unwrap_or_default();
-                for folder in &state.workspace.folders {
-                    if let Ok(folder) = folder.to_file_path() {
-                        if let Some(folder) = folder.to_str() {
-                            if path.starts_with(folder) {
-                                path = &path[folder.len() + 1..];
-                                break;
-                            }
+                // Add some metadata about where the completion was found
+                let mut path = uri.as_str().to_owned();
+
+                if uri.scheme() == "file" {
+                    let Ok(file_path) = uri.to_file_path() else {
+                        return;
+                    };
```

Comment:
Isn't this a change in behavior? i.e. it's now saying we dont push the completion if we can't figure out where it came from, where previously i think we just pushed it anyways with that default `path`?